Routes patch

paulogdm opened the pull request:
This patch fix the routes configuration in `now.json`.

paulogdm committed:
Routes patch

ericadamski reviewed (approved):
Is this just using regular expressions like from the documentation here: https://zeit.co/docs/v2/deployments/routes/#route-parameters

paulogdm commented:
Exactly! In my case, I'm using `$1` which is a short for `match the first group`. You can use regex101.com to see the matching groups...

paulogdm commented:
Example: `[{ "src": "/product/(?<id>[^/]+)", "dest": "/product?id=$id" }]` is a short for `take any character unless it is / and name this group "id"`. And in the destination we are using this match group.

paulogdm committed:
Update now.json

When using `mdx-deck` any route needs to fallback to `index.html`
Check: https://github.com/jxnblk/mdx-deck/pull/274/files
